engine/src/cmd/basecomputer.cpp

```diff
@@ -3942,7 +3943,7 @@ string buildUpgradeDescription(Cargo &item) {
     const std::string key = item.GetName() + "__upgrades";
     PyObject* args = PyTuple_Pack(1, PyUnicode_FromString(key.c_str()));
     const std::string text = GetString("get_upgrade_info", "upgrade_view",
-        "python/base_computer/upgrade_view.py", args);
+        "upgrade_view.py", args);
```

Comment:
Can you expand on why you did this?

Reply:
You were specifying the subfolder (`"python/base_computer"`) twice. Once here, and once in the Python path initialization code. That's part of why VS was failing to locate these Python scripts.

Now the subfolder is specified in only one place: `initpaths()`. Which is more or less in keeping with the way existing code did things.

Reply:
I'm concerned that we could potentially have two utils.py somewhere with unexpected and hard to debug errors.